calculator/main.py

```diff
@@ -12,8 +12,15 @@ def create_new_calculator(operations=None):
 
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
+                       
+    (Pseudocode added by jon)
+    Returns: a dict containing:
+        a dict named 'operations' with their functions.
+        a list named 'history' of tuples. [should initialize empty.]
     """
-    pass
+    if not operations: 
+        operations = {}
+    return {'operations': operations, 'history': []}
 
```

Comment:
a shorter way to do this would be to change the default value of operations from None to {}. this way you dont have to check whether operations is empty and can just return it